packages/kit/src/template.ts

```diff
@@ -78,5 +78,5 @@ export function normalizeTemplate (template: NuxtTemplate<any> | string): Resolv
  * You can pass a filter within the options to selectively regenerate a subset of templates.
  */
 export function updateTemplates (options?: { filter?: (template: ResolvedNuxtTemplate<any>) => boolean }) {
-  return useNuxt().hooks.callHook('builder:generateApp', options)
+  return tryUseNuxt()?.hooks.callHook('builder:generateApp', options)
```

Comment:
Previously `callHook` always returned a Promise. While this is small, i think would be nice if we use `async/await` to make sure return type is promise (`?.` leads to `undefined`)